index.js

```diff
@@ -4,15 +4,15 @@
  */
 module.exports = {
     // Core libraries
-    I13nNode: require('./dist/libs/I13nNode'),
```

Comment:
https://babeljs.io/docs/en/v7-migration-api#export-changes

> Dropped use of add-module-exports plugin on Babel packages. This had to be used earlier to prevent a breaking change with our exports. If you import a Babel package in a library you may need to use .default when using require rather than import.